build.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
I think the name and the location of this file could be confusing to users? They could mistake it as a script for building the framework itself?

Reply:
Moved!